### PR TITLE
Clean up the DocFx javascript and css.

### PR DESCRIPTION
**Pull request by alextrepa**
GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?
- Code style update (formatting)
- Refactoring (no functional changes, no api changes)
- Documentation content changes

## What is the current behavior?

DocFx has been modded for the needs of Uno. Those changes were all made in a single Js and scss file. It was becoming increasingly difficult to maintain.

## What is the new behavior?

The JavaScript and the scss have been split by components, the project is easier to run locally, and the documentation to run it has been updated. Maintaining and adding functionality should be easier.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] ~[Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)~
- [ ] ~Validated PR `Screenshots Compare Test Run` results.~
- [x] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

https://dev.azure.com/nventive/UNO%20Marketing/_workitems/edit/227813

### CODE REVIEW

**Review by nv-jbleau — APPROVED** [review]

File: doc/ .browserlistrc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Browsers that we support
```
Comment (Youssef1313) [inline doc/ .browserlistrc:1]: Is the space in file name intentional?

File: doc/README.md [hunk 2]
```diff
@@ -2,7 +2,9 @@
 
 This folder contains source code for the generation of uno's documentation
 
-# Running a local environement
+> ! It's very important that you read the deploy section before commiting anything to the repo.
```
Comment (Youssef1313) [inline doc/README.md:5]:
```suggestion
> [!IMPORTANT]
> It's very important that you read the deploy section before commiting anything to the repo.
```

File: doc/ .browserlistrc (hunk 1 above)
Comment (alextrepa) [reply doc/ .browserlistrc:1]: Unless I'm missing something, yes it is.

Comment (carldebilly) [reply doc/ .browserlistrc:1]: @alextrepa I agree with Youssef: that's very odd.
image

Comment (alextrepa) [reply doc/ .browserlistrc]: Yeah, you're right, it's fixed

**Review by davidjohnoliver — APPROVED** [review]

**Pull request merged**